yarn.lock

```diff
@@ -0,0 +1,7333 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
If you remove the `yarn.lock` file and add it to the project's `.gitignore` we can use that going forward until we're ready to migrate to yarn.

TLDR: Yes, do it, it won't mess up deploy.